website/public/js/controllers/notificationCtrl.js

```diff
@@ -121,17 +128,6 @@ habitrpg.controller('NotificationCtrl',
       $rootScope.openModal('achievements/contributor');
     });
 
-    /*_.each(['weapon', 'head', 'chest', 'shield'], function(watched){
```

Comment:
You're like a one-man angry mob, calling for the death of various things that displease you!